app/views/providers/references/edit.html.erb

```diff
@@ -0,0 +1,32 @@
+<% page_title = t("helpers.label.provider.references") %>
+
+<%= content_for :page_title, title_with_error_prefix(page_title, @form_object.errors.present?) %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(details_provider_recruitment_cycle_path(@provider.provider_code, @provider.recruitment_cycle_year)) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(
+      model: @form_object,
+      url: provider_recruitment_cycle_references_path(@provider.provider_code, @provider.recruitment_cycle_year),
+      method: :post,
+      builder: GOVUKDesignSystemFormBuilder::FormBuilder,
+    ) do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l"><%= @provider.provider_name %></span>
```

Comment:
Ah, okay. In that case, just ensure this page has a heading:

```suggestion
        <span class="govuk-caption-l"><%= @provider.provider_name %></span>
        <%= page_title %>
```

Reply:
If the form only displays one field, should we still display the head "Provider reference numbers"?